app/models/component.rb

```diff
@@ -137,7 +138,7 @@ def from_mapping(benchmark)
 
   def largest_rule_id
     # rule_id is a string, convert it to a number and then extract the current highest number.
-    Rule.connection.execute("SELECT MAX(TO_NUMBER(rule_id, '999999')) FROM rules
+    Rule.connection.execute("SELECT MAX(TO_NUMBER(rule_id, '999999')) FROM base_rules
```

Comment:
So this is likely because this PR is not backwards compatible with the previous database. In order for this PR to work fully you'll need to drop your database and create it on this PR. Once we have reached a V1.0 release we'll have to stop breaking existing installations.